src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -1030,7 +1030,7 @@ private void ImportDataMembers()
                         DataMember memberContract = new DataMember(member);
                         if (property != null)
                         {
-                            MethodInfo getMethod = property.GetMethod;
+                            MethodInfo getMethod = property.GetGetMethod();
```

Comment:
I had to look at the implementation of GetMethod and GetGetMethod to understand that they were different.  Subtle. Could not have told you that they were the same except for visibility.  Learn something new every day :)